production/ksonnet/loki/index-gateway.libsonnet

```diff
@@ -8,12 +8,12 @@
       storage_config+: if $._config.use_index_gateway then {
         boltdb_shipper+: {
           index_gateway_client+: {
-            server_address: 'dns:///index-gateway.%s.svc.cluster.local:9095' % $._config.namespace,
+            server_address: 'dns+index-gateway-headless.%s.svc.cluster.local:9095' % $._config.namespace,
```

Comment:
I'm using Loki 2.9.2, and wanted to test out the tsdb shipper, so I imported 1d172a7eeb851f9939ad1d6e40c54e1686ace8c2 using jsonnet-bundler to get the shipper fix in `production/ksonnet/loki/shipper.libsonnet`. This `server_address` change broke the server, and I can't find any docs to explain the `dns+` prefix. Was this change intended?

Reply:
Nevermind, I found https://github.com/grafana/loki/pull/10783